customers-service-reactive/src/main/java/com/petclinic/customersservice/customersExceptions/http/HttpErrorInfo.java

```diff
@@ -25,20 +27,4 @@ public HttpErrorInfo() {
         this.message = null;
     }
 
-    public ZonedDateTime getTimestamp() {
```

Comment:
In all of our projects with Christine, we always have this usually so I'm not sure whether or not this is a 100% prerequisite but definitely something I'd keep for exception handling, but that's just me...

Reply:
```
  public ZonedDateTime getTimestamp() {
        return this.timestamp;
    }
```
There is a method that pulls a timestamp

Reply:
looks good then, brother